rspace/src/test/scala/coop/rchain/rspace/test/InMemoryStore.scala

```diff
@@ -24,10 +23,15 @@ class InMemoryStore[C, P, A, K] private (
 
   val eventsCounter: StoreEventsCounter = new StoreEventsCounter()
 
+  var _dbGNATs: Map[Blake2b256Hash, GNAT[C, P, A, K]] = dbGNATs
```

Comment:
Why not just do `Map.empty` here and remove the class parameters list? 
BTW, shouldn't the `var` be `private[this]`?